Program.cs

```diff
@@ -257,43 +259,82 @@ public static void CombineAtlas(string inputPath)
 
         static void Main(string[] args)
         {
-            string input = string.Empty;
+            string input  = "";
```

Comment:
string.Empty is the preferred way to make a blank string here it's the same thing as ""

Reply:
Alrighty, although my research indicated that there isn't much of a real difference between the two. I mainly just wanted to make them both consistent, since one was `string.Empty` and the other was `""`